cmd/krew/cmd/util.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
```

Comment:
Should this go in `cmd/internal`? It seems like `cmd` only has the cobra commands, so it could be misleading at first glance where devs could think there's a `krew util` subcommand. This is super minor and probably just for my own clarification.

Reply:
possibly, though, it's kinda unidiomatic to make util packages for these sort of things. You're right that this almost implies that there's a "util" command; though, do we care about that?

Reply:
Might rename to namingutils.go

Reply:
I don't think its a big deal, just a small nit from my perspective.

Reply:
renamed to namingutils.go